Cubical/Foundations/GroupoidLaws.agda

```diff
@@ -232,6 +232,6 @@ doubleCompPath-elim' p q r = (split-leftright' p q r) ∙ (sym (leftright p (q 
 
 -- deducing associativity for compPath
 
--- compPath-assoc : {ℓ : Level} {A : Set ℓ} {w x y z : A} (p : w ≡ x) (q : x ≡ y) (r : y ≡ z) →
---                  (p ∙ q) ∙ r ≡ p ∙ (q ∙ r)
--- compPath-assoc p q r = (sym (doubleCompPath-elim p q r)) ∙ (doubleCompPath-elim' p q r)
+∙-assoc : {ℓ : Level} {A : Set ℓ} {w x y z : A} (p : w ≡ x) (q : x ≡ y) (r : y ≡ z) →
```

Comment:
This is already defined in that file, but it is called `assoc`. Can you please comment this one out and rename the other one?